Objects/Objects/Structural/Material/Timber.cs

```diff
@@ -6,28 +6,31 @@
 
 namespace Objects.Structural.Materials
 {
-    public class Timber : Material
+  public class Timber : Material
+  {
+    //missing timber-specific properties? parallel to grain, perpendicular to grain
+    public string species { get; set; }
+    public Timber() 
     {
-        //missing timber-specific properties? parallel to grain, perpendicular to grain 
-        public string species { get; set; }
```

Comment:
You need to still keep an empty constructor. Speckle needs an empty constructor.